### ISSUE TITLE
Multiple Loss Function

### ISSUE THREAD

**Issue by bluesky314** [junfu1115/DANet#96]
In SegmentationLosses at https://github.com/junfu1115/DANet/blob/97f0c4224ab37084d418f9f3acc8d9a3b86373df/encoding/nn/customize.py

For the fully auxillary loss function we have terms such as 
se_target = self._get_batch_label_vector(target, nclass=self.nclass).type_as(pred1)
is loss3 = self.bceloss(F.sigmoid(se_pred), se_target)

This is not a regular cross-entropy term but some kind of histogram comparsion. There is no mention of this in the paper. Can we get some clarity as to what is happening here and why it is only used for one of the outputs?
